superduperdb/components/template.py

```diff
@@ -89,7 +118,16 @@ def export(self, path: str, format: str = 'json'):
         |_files/*
         ```
         """
+        # Move the component leaves and blobs to the top level
         r = self.dict().encode()
+        component_leaves = r.pop('_component_leaves', {})
+        component_blobs = r.pop('_component_blobs', {})
+        component_leaves_keys = list(component_leaves.keys())
+        component_blobs_keys = list(component_blobs.keys())
+        r['_leaves'].update(component_leaves)
+        r['_blobs'].update(component_blobs)
+        r['_component_leaves_keys'] = component_leaves_keys
+        r['_component_blobs_keys'] = component_blobs_keys
```

Comment:
@blythed Please check the template section. If possible, roll back and help fix it. Our new decoding process has introduced some bugs in the template.

Reason: We stopped decoding the component because we only need to retain the dictionary information of the component. However, the decoding logic is missing some _leaves.